app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php

```diff
@@ -66,7 +66,6 @@ public function execute(ContextInterface $context, CartInterface $cart, array $s
         }
 
         if (null === $customerAddressId) {
-            $addressInput['country_code'] = strtoupper($addressInput['country_code']);
```

Comment:
We should preserve this behaviour.

Reply:
I will transfer this cod to the factory. In this case, it works only for the shipping address.